source/standards/pair-programming.html.md.erb

```diff
@@ -0,0 +1,81 @@
+We provide some of the following advice to help pairs work effectively together.
+
+### Rotate pairs regularly
+Pairs should rotate regularly at a cadence that makes sense for the team.
+This might be after a single piece of work or after a fixed period of time.
+Rotating pairs spreads knowledge, increases team resilience, and builds ownership of the codebase.
+
+### Discuss how to pair together
+Pairs should agree on the way that they will pair before starting the work itself.
+Your pair might favour ping-pong programming (where one person writes a test, the other person makes it pass and then writes the next test) or strong-style pair programming (your ideas need to enter the codebase through the other person’s input).
+This also includes agreeing when to take breaks.
+
+### Take breaks
+Working together and staying focused for extended periods of time can be tiring.
+Pairs should make sure that they take regular breaks and where possible work in small iterative batches rather than large changes.
+You might find the Pomodoro technique useful or one of the many pair/mob programming timers available online.
+Have a debrief at the end of the day if it would be helpful, and raise any recurring issues at your team’s retrospectives or with your tech lead.
+
+### Pair where appropriate
+Pairing is not suitable for every type of work, so use your discretion within the team.
+You might choose not to pair on automated refactorings or work that would be described as Clear within the Cynefin model.
+Remember that developers are people and as such might be unable to pair on any given day for their own personal reasons. Be respectful and understanding if someone is unable to pair.
+
+### Communicate effectively
+Pair programming is both a social skill and a technological skill.
+It works most effectively in an environment where the pair respect each other's skills and needs, and communicate with openness, empathy, and honesty.
+People in teams that have embraced concepts like [psychological safety](https://en.wikipedia.org/wiki/Psychological_safety) and [non-violent communication](https://www.cnvc.org/) are likely to be well-equipped for pair programming.
+
+## Reviewing code as a pair
+
+The standard approach to code review in GDS is that teams should use a pull-request (PR) process to ensure that all code is reviewed by a second pair of eyes.
+Some teams allow PRs worked on in a pair to be approved by one of the members of the pair, rather than ask for a review from an external third person.
+
+This has several benefits, such as:
+
+- speeding up the flow of work.
+- enabling the pair to move on to another task without context switching.
+- avoiding a third person having to context switch to review the code.
+
+However, teams should adjust their policy on this approach depending on the risk tolerance of their team, service, or programme.
```

Comment:
I think some of this section duplicates content in 'Benefits' above. 

I think it might make sense to merge the content about the benefits of working this way into the Benefits section, and to focus on the stuff around checking that this meets your local policies here.

Reply:
Good point, I'll make some changes to move things around

Reply:
I've consolidated the code review section to remove duplication, and moved some of the content from the "Benefits" section down at the same time.

Reply:
Good change!